justfile

```diff
@@ -29,7 +29,12 @@ dep-sort:
 doc:
     cargo doc --no-deps --document-private-items
 
+release version:
+    chmod +x scripts/release.sh
+    scripts/release.sh {{version}}
```

Comment:
Wondering if just doing `sh scripts/release.sh` would avoid the `chmod +x`

Reply:
probably, but isn't it easiest to just maintain the file as executable?

Reply:
Yeah, I think git will just store the executable bit.